### DISCUSSION TITLE
Google Ads report "Evasive ad content" and 'Malicious Software" on my Publii site

### DISCUSSION THREAD

**Discussion by briff** [GetPublii/Publii#1560] [Help]
Does anyone have experience with this? Google Ads doesn't allow me to create ads for my publii site, because of "Evasive ad content" and "Malicious software". 
I checked with Google Search console and other tools, turned out clear, so I have no idea what the problem is (and Google won't tell me). 
The site is served from an Amazon S3 bucket via CloudFront, and no spurious files are uploaded, so the site is not hacked.

**Comment by bobmitro** [comment]
Any Url?

**Reply by briff** [reply]
The URL in ROT13 encoding is uggcf://fhylnanaqern.uh (I don't want to post it here directly to not mess up SEO).

**Reply by bobmitro** [reply]
I checked your site with several tools and scenarios, and none of them detected any issues. I suspect that you might have used some scripts in the past that Google might find problematic. I'm not an expert in Google Ads, but I'm confident that the Publii site itself is not the cause of these messages.
